crates/sui-framework/packages/bridge/sources/limiter.move

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module bridge::limiter {
+
+    use std::option;
+    use std::type_name;
+    use std::type_name::TypeName;
+    use std::vector;
+    use sui::clock;
+    use sui::clock::Clock;
+    use sui::math::pow;
+    use sui::vec_map;
+    use sui::vec_map::VecMap;
+    use bridge::treasury;
+    use bridge::chain_ids::BridgeRoute;
+    #[test_only]
+    use sui::test_scenario;
+    #[test_only]
+    use sui::test_utils::destroy;
+    #[test_only]
+    use bridge::chain_ids;
+    #[test_only]
+    use bridge::eth::ETH;
+
+    const ETransferAmountExceedLimit: u64 = 0;
+
+    // TODO: Make this configurable?
+    const DEFAULT_TRANSFER_LIMIT: u64 = 0;
+
+    struct TransferLimiter has store {
+        transfer_limits: VecMap<BridgeRoute, u64>,
+        // USD notional value, 0 DP accuracy
+        notional_values: VecMap<TypeName, u64>,
+        // Per hour transfer amount for each bridge route
+        transfer_records: VecMap<BridgeRoute, TransferRecord>,
+    }
+
+    struct TransferRecord has store {
+        hour_head: u64,
+        hour_tail: u64,
+        per_hour_amounts: vector<u64>,
+        // total amount in USD, 0 DP accuracy
+        total_amount: u64
+    }
+
+    public fun new(): TransferLimiter {
+        TransferLimiter {
+            transfer_limits: vec_map::empty(),
+            notional_values: vec_map::empty(),
+            transfer_records: vec_map::empty()
```

Comment:
sorry for not mentioning this in the previous commit, but how do we init the limit initially? Should we hardcode it here so we don't need validator sigs?

Reply:
ya we can hard code some initial values if we want to avoid sig at the beginning, do we know what is the "good" limits to set?

Reply:
let's say 5M? we need some consultation on these numbers before launch